ui/datasets/semantic-ui/js/datasets/search/ResultsListItem.jsx

```diff
@@ -1,82 +1,265 @@
+                <a href={result?.links?.self_html}>{title}</a>
+              </Item.Header>
+              <Item.Meta className="rel-mt-1">
+                <Creatibutors creatibutors={creatibutors} />
+                <Label.Group className="rel-mt-1">
+                  {subjects.map((subject, index) => (
+                    <Label key={index}>{subject.title.value}</Label>
+                  ))}
+                </Label.Group>
+              </Item.Meta>
+              {abstract && (
+                <Item.Description
+                  className="rel-mt-1"
+                  dangerouslySetInnerHTML={{
+                    __html: sanitizeHtml(abstract, {
+                      allowedTags: allowedHtmlTags,
+                      allowedAttributes: {},
+                    }),
+                  }}
+                />
+              )}
+              <Item.Extra className="rel-mt-1">
+                <p>
+                  {publicationDate && <span>{publicationDate}</span>}
+                  {originalRepositories?.map((originalRepository, index) => (
+                    <p className="rel-ml-1" key={originalRepository.id}>
+                      <span>{originalRepository}</span>
+                      {index < originalRepositories.length - 1 ? ", " : null}
+                    </p>
+                  ))}
+                  {language && (
+                    <span className="rel-ml-1">{language.toUpperCase()}</span>
+                  )}
+                </p>
+              </Item.Extra>
+            </Grid.Column>
+
+            <Grid.Column width={3}>
+              <List>
+                <List.Item>This section is unclear</List.Item>
+                <List.Item>
+                  <Label>License</Label>
+                </List.Item>
+                <List.Item>
+                  <Label>Počet souborů</Label>
+                </List.Item>
+                <List.Item>
+                  <Label>Verze: {result.metadata?.version}</Label>
+                </List.Item>
+                <List.Item>
+                  <Label>Typ souboru</Label>
+                </List.Item>
+                <List.Item>
+                  <Label>Originál záznam</Label>
+                </List.Item>
+              </List>
+            </Grid.Column>
+          </Grid.Row>
+        </Grid>
+      </Item.Content>
+    </Item>
+  );
+};
+
+ResultsListItemComponent.propTypes = {
+  result: PropTypes.object.isRequired,
+};
+
+export default ResultsListItemComponent;
+
+export const IconIdentifier = ({ link, badgeTitle, icon, alt, className }) => {
+  return link ? (
+    <span className={`creatibutor-identifier ${className}`}>
+      <a
+        className="no-text-decoration mr-0"
+        href={link}
+        aria-label={badgeTitle}
+        title={badgeTitle}
+        key={link}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <Image
+          className="inline-id-icon identifier-badge inline"
+          src={icon}
```

Comment:
Placeholder if unsupported scheme? Even better use identifier component from oarepo-ui?

Reply:
I think we can do it now :)